src/VecSim/algorithms/hnsw/hnsw_factory.cpp

```diff
@@ -1,22 +1,20 @@
 #include "VecSim/algorithms/hnsw/hnsw_factory.h"
-#include "VecSim/algorithms/hnsw/hnsw_wrapper.h"
-#include "VecSim/algorithms/hnsw/hnswlib.h" //linklistsizeint
-#include "VecSim/vec_sim_common.h"          // labelType
+#include "VecSim/algorithms/hnsw/hnsw.h" //linklistsizeint
```

Comment:
i find them helping where the include purpose is not obvious :sweat_smile: 
here it might be redundant (I guess leftovers. hnswlib was included only for the linklistsizeint typedef)